app/src/org/commcare/android/view/SquareButtonWithNotification.java

```diff
@@ -13,53 +13,56 @@
 import org.commcare.dalvik.R;
 
 /**
+ * Square button with subtext and notification text
+ *
  * @author Daniel Luna (dcluna@dimagi.com)
  */
 public class SquareButtonWithNotification extends RelativeLayout {
```

Comment:
I think it is possible but I can't really figure out how to do it. There is some sorta funky stuff going on with nesting styleables (https://github.com/dimagi/commcare-odk/blob/refactor/text-image-button-restructure/app/res/values/attrs.xml#L33-L42) that would have to be ironed out. I'm going to differ this to a later date.